WcaOnRails/config/locales/en.yml

```diff
@@ -280,6 +280,15 @@ en:
       #context: a vote
       vote:
         comment: "Comment"
+      #context: an incident
+      incident:
```

Comment:
Seeing these things get translated reminds me of #1840. Maybe it would be good to either not translate these fields, or include a message about filling them out in English?

Reply:
The WRC will fill the fields so I guess we won't run into trouble, but just in case I'll add the disclaimer there too!